spring-native/src/main/java/org/springframework/aot/test/AotContextLoader.java

```diff
@@ -70,6 +77,24 @@
 		}
 	}
 
+	@SuppressWarnings("unchecked")
```

Comment:
Isn't there a case to refactor the existing method to pass the method name and expected type? The cast is happening anyway so I don't see a lot of benefits in having two methods for the reflection business.

Reply:
I was actually considering doing that but hadn't gotten to it.

It's done now (locally).